src/Views/HardwareView.vala

```diff
@@ -280,6 +282,23 @@ public class About.HardwareView : Gtk.Grid {
 
         return 0;
     }
+
+    private string get_storage_type () {
+        string storage = "";
+        try {
+            UDisks.DriveSkeleton drive = new UDisks.DriveSkeleton ();
```

Comment:
elementary code style here is to use `var drive` instead of `UDisks.DriveSkeleton drive`